applications/physics/ICF/pre_train_jag_wae.py

```diff
@@ -90,7 +149,7 @@ def construct_model():
 if __name__ == '__main__':
     import lbann
     
-    trainer = lbann.Trainer()
+    trainer = lbann.Trainer(procs_per_trainer=args.procs_per_trainer)
```

Comment:
I believe that this model also needs the seriealize_io flag.  The only time that we don't need it is if we are preloading the data store, but even then we should just set it.

Reply:
This model (and the next one) has preload_data_store, so it is the reason I didnt set as am not sure how preload_data_store/use_data_store interacts with serialize_io. In any case, I will set it for all models.

Reply:
I have made _serialize_io=True_ default.